dspace-api/src/test/data/dspaceFolder/assetstore/sciello-single-record.txt

```diff
@@ -0,0 +1,24 @@
+TY  - JOUR
```

Comment:
All test records should be moved alongside the test classes.  So, this should be moved to `dspace-server-webapp/src/test/resources/org/dspace/external/provider/impl` alongside all other test data responses.

Also, this file has a misspelling in the name. It should say `scielo`, not `sciello`